src/detectors/DRICH/CMakeLists.txt

```
@@ -17,3 +17,6 @@ plugin_glob_all(${PLUGIN_NAME})
 plugin_add_irt(${PLUGIN_NAME})
 plugin_add_event_model(${PLUGIN_NAME})
 plugin_add_dd4hep(${PLUGIN_NAME})
+
+# Add libraries
+plugin_add_acts(${PLUGIN_NAME})
```

No need to separate it from the other `plugin_add`s
```suggestion
plugin_add_acts(${PLUGIN_NAME})
```